src/main/java/bio/terra/app/controller/SnapshotsApiController.java

```diff
@@ -65,7 +66,7 @@ public class SnapshotsApiController implements SnapshotsApi {
 
   // We do not include Access_Information since it can get expensive, and for backwards compat
```

Comment:
Flag for reviewers: re: "backwards compat", do we really not want to introduce new default fields for snapshot retrieval?  I can remove `DUOS` as a default if so.

Reply:
Eh, I think it's fine to add fields like this